README.md

```diff
@@ -20,23 +22,29 @@ There are two ways to download, install and run this Web Service:
 
  To run the app as part of the standard Companies House Vagrant set-up, follow [these set-up instructions](https://github.com/companieshouse/vagrant-chs-development-v2) on how to run Vagrant on your machine
 
- Once Vagrant is up and running, `ssh` into Vagrant by running:
+ Once Vagrant is up and running you can either start the service on its own:
+
  ```
-vagrant ssh chs-dev
+ ubic start chs.dev-hub.applications-developer-service
  ```
 
-Now run the `ubic status` command to see what services are running.
+ Or you start it up as part of the chs dev-hub group (you will need to do this in order to be able to use the api functionality properly):
 
-If no services are running, you can start them by typing the command:
-```
-ubic start chs && ubic start applications.developer.web.ch.gov.uk
-```
+ ```
+ ubic start chs.dev-hub
+ ```
+You should also make sure that the account service, cdn and path router are running in chs core:
+
+- `ubic start chs.core.account`
+- `ubic start chs.core.cdn`
+- `ubic start chs.core.path-router`
+
+Or you may prefer to simply start up all of chs.core:
 
-If services are running but patchy, you can restart them by typing the command:
 ```
-ubic restart chs && ubic restart applications.developer.web.ch.gov.uk
+ubic start chs.core
 ```
-After a short wait, the Web Service should be accessible in your browser at: http://web.chs-dev.internal:18555/
+Once the necessary applications have started up, the service should be accessible in you browser at: [http://dev.chs-dev.internal:4904](http://dev.chs-dev.internal:4904).

```

Comment:
Are we repeating the link for a reason?

Reply:
Whereabaouts?

Reply:
[http://dev.chs-dev.internal:4904](http://dev.chs-dev.internal:4904).

This might just be me not understanding the brackets and stuff?

Reply:
Although now I'm looking into it, that might not be necessary when the text I want to display is the link itself....

Reply:
Also, the link is wrong - applications web is on port 4100

Reply:
This will need to be changed in the other places where this link is referenced

Reply:
Changed